deploy/helm/charts/README.md

```diff
@@ -55,39 +57,53 @@ _See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command doc
 
 ```console
 # Helm
-$ helm upgrade [RELEASE_NAME] [CHART] --install
+helm upgrade [RELEASE_NAME] [CHART] --install --namespace [NAMESPACE]
```

Comment:
q_: do we need to provide the --install in upgrade command as well?

Reply:
Nope. Removing it.

Reply:
We don't need it. Keeping it as per discussion.